README.md

```diff
@@ -306,6 +311,32 @@ Here's an example of what the notifications look like:
 ![Mattermost notifications](.github/assets/mattermost-alerts.png)
 
 
+#### Configuring Messagebird alerts
+
+Example of sending **sms** message alert by using Messagebird
+
+```yaml
+alerting:
+  messagebird:
+    access-key: "..."
+    originator: "31619191918"
+    recipients: "31619191919"
```

Comment:
If you could add another recipient to the `recipients` field, that'd make it obvious that you can specify one or more recipients